gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/AuthConfig.groovy

```diff
@@ -41,11 +40,19 @@ class AuthConfig {
   }
 
   static void configure(HttpSecurity http) throws Exception {
-    http.csrf().disable()
-    http.authorizeRequests()
+    http
+      .authorizeRequests()
         .antMatchers(HttpMethod.OPTIONS, "/**").permitAll()
         .antMatchers('/auth/user').permitAll()
         .antMatchers('/health').permitAll()
         .antMatchers('/**').authenticated()
+        .and()
+      .logout()
+        .logoutUrl("/auth/logout")
```

Comment:
The request gets matched to the logout filter. By default it's `/logout`, but I'd like to move everything related to auth under `/auth`, so I overrode it.